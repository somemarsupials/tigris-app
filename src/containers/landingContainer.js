import React, { Component } from 'react';
import axios from 'axios';
import { LandingPage } from '../components/pages';

export class LandingContainer extends Component {
  async componentDidMount() {
    axios.get({
    });
  };

  render() {
    return (<LandingPage {...this.props} />);
  };
};
