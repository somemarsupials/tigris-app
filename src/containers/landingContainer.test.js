import React from 'react';
import { shallow } from 'enzyme';
import { LandingContainer } from './landingContainer';

describe('LandingContainer', () => {
  it('matches snapshot', () => {
    expect(shallow(<LandingContainer />)).toMatchSnapshot();
  });
});



