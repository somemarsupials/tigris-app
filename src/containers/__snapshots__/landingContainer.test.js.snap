// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LandingContainer matches snapshot 1`] = `<_default />`;
